src/collections/events/service-mesh-and-meshery/index.mdx

```diff
@@ -0,0 +1,26 @@
+---
+title: "Journey Into the World of Service Meshes and Meshery"
+date: 2022-01-29
+thumbnail: ./service-mesh-and-meshery.png
+eurl: https://sched.co/siEs
+published: true
+upcoming: true
+speakers: ["Nithish Karthik", "Aditya Krishna Sharma"]
```

Comment:
@Prakhar-Agarwal-byte these name should match existing contributors names such that the name becomes a hyperlink to the community member profile. Do you know if that was the case when you built it?

Reply:
I copied the names from the issue itself. The name mentioned in [contributor profile](https://61bd62287cbfaf9cc2dcbf78--layer5ng.netlify.app/community/members/adithya-krishna) is "Adithya Krishna"

Reply:
@Prakhar-Agarwal-byte You can change it to the one mentioned in the profile itself, no problem :)